Mixpanel/Mixpanel.m

```diff
@@ -856,6 +962,11 @@ - (NSString *)eventBindingsFilePath
     return [self filePathFor:@"event_bindings"];
 }
 
+- (NSString *)optOutFilePath
+{
+    return [self filePathFor:@"optOut"];
```

Comment:
so we don't create a new file for each param, we could use https://github.com/mixpanel/mixpanel-iphone/blob/master/Mixpanel/AutomaticEvents.m#L26 instead
ideally, some refactoring of the local storage would be amazing since it's all over the place right now